drizzlepac/devutils/analyze_mvm_gaia_alignment.py

```diff
@@ -102,60 +228,43 @@ def perform(mosaic_imgname, flcflt_list, diagnostic_mode=False, log_level=loguti
-                                                def_fwhm=3.0, box_size=7, block_size=(1024, 1024),
-                                                diagnostic_mode=False)
-    # mfwhm = 25.0 # If it fails due to a lack of sources, use mfwhm = 25.0 instead.
-    daofind = decutils.UserStarFinder(fwhm=mfwhm, threshold=0.0, coords=xy_gaia_coords,
-                                      sharplo=0.4, sharphi=0.9)
-    detection_table = daofind(mosaic_hdu["SCI"].data, mask=dao_mask_array)
-    detection_table.rename_column('xcentroid', 'X')
-    detection_table.rename_column('ycentroid', 'Y')
-    n_detection = len(detection_table)
-    n_gaia = len(gaia_table)
-    pct_detection = 100.0 * (float(n_detection) / float(n_gaia))
-    log.info("Found {} peaks from {} GAIA source(s)".format(n_detection, n_gaia))
-    log.info("{}% of GAIA sources detected".format(pct_detection))
-
-    # 4: convert daostarfinder output x, y centroid positions to RA, DEC using step 1 WCS info
-    ra, dec = mosaic_wcs.all_pix2world(detection_table['X'], detection_table['Y'], 0)
-    ra_col = Column(name="RA", data=ra, dtype=np.float64)
-    dec_col = Column(name="DEC", data=dec, dtype=np.float64)
-    detection_table.add_columns([ra_col, dec_col], indexes=[3, 3])
-    if diagnostic_mode:
-        write_region_file("test_detection.reg", detection_table, ['RA', 'DEC'], log_level=log_level)
-
-    # 5: Identify and isolate X, Y, RA and DEC values common to both the gaia and detection tables.
-    # 5a: find sources common to both the gaia table and the detection table
-    try:
-        coo_prefix_string = ''.join(random.choice(string.ascii_lowercase + string.digits) for _ in range(6))
-        gaia_coo_filename = "{}_gaia.coo".format(coo_prefix_string)
-        det_coo_filename = "{}_det.coo".format(coo_prefix_string)
-        write_region_file(gaia_coo_filename, gaia_table, ['X', 'Y'], verbose=False)
-        write_region_file(det_coo_filename, detection_table, ['X', 'Y'], verbose=False)
-        matches_gaia_to_det, matches_det_to_gaia = cu.getMatchedLists([gaia_coo_filename, det_coo_filename],
-                                                                      [mosaic_imgname, mosaic_imgname],
-                                                                      [n_gaia, n_detection],
-                                                                      log_level)
-        if len(matches_gaia_to_det) == 0:
-            err_msg = "Error: No matching sources found."
-            log.error(err_msg)
-            raise Exception(err_msg)
-    finally:
-        for item in [det_coo_filename, gaia_coo_filename]:
-            if os.path.exists(item):
-                log.debug("Removing temp coord file {}".format(item))
-                os.remove(item)
-
+    # 4: compute FWHM for source finding based on sources in image
+    mpeaks, fwhm = decutils.find_point_sources(mosaic_imgname, mask=np.invert(dao_mask_array),
```

Comment:
You need to force 'dao_mask_array' to be bool for 'np.invert' to work correctly, then pass the result as an np.int16 array for the masking to work correctly.  Inverting an np.int array results in negative values.  It can be done using the following suggestion: 
```suggestion
    mpeaks, fwhm = decutils.find_point_sources(mosaic_imgname, mask=np.invert(dao_mask_array.astype(bool)).astype(np.int16),
```

Reply:
issue resolved by commit [819e7f0](https://github.com/spacetelescope/drizzlepac/pull/1241/commits/819e7f06bfd0e684d383ba0c216da21a5608e92b)